src/controls/shared.js

```diff
@@ -1,17 +1,14 @@
 import PropTypes from "prop-types";
 import styled, { css } from "styled-components";
 
-const PILL_HEIGHT = 32;
+import { PillContainer, PillValue } from "../pill";
```

Comment:
Since all of these controls were "pills" I created a new `Pill` component so that I could reuse these exact styles for the pill links in the Header.

Reply:
hell yeah